package.json

```diff
@@ -46,11 +46,11 @@
     "resin-cli-visuals": "^1.2.8",
     "resin-discoverable-services": "git+https://github.com/resin-io-modules/resin-discoverable-services#find-on-all-interfaces",
     "resin-sdk-preconfigured": "^6.4.0",
+    "resin-semver": "^0.12.0",
```

Comment:
Totally incidental, but I think we might be getting close to a 1.0.0 release here...

Reply:
In my mind, the 1.0.0 release will happen once we have coverage for every `semver` method.